lang/java/avro/src/main/java/org/apache/avro/file/DataFileStream.java

```diff
@@ -102,7 +102,7 @@ void initialize(InputStream in) throws IOException {
       throw new IOException("Not an Avro data file.", e);
```

Comment:
Is there an analogous case here to the EOF above? If there weren't enough bytes in the stream for the magic check, right now here we throw a bare IOE, right?